maintainers/working_groups/learning_journey/documentation-survey.md

```diff
@@ -0,0 +1,108 @@
+# Documentation survey
+
+This is a survey of materials put together by @fricklerhandwerk
```

Comment:
```suggestion
```

That doesn't matter, it's a collaborative document. Put me as a co-author of the commit, it's enough attribution.